web/app/components/header/account-setting/model-provider-page/model-modal/index.tsx

```diff
@@ -205,7 +205,7 @@ const ModelModal: FC<ModelModalProps> = ({
   const encodeSecretValues = useCallback((v: FormValue) => {
     const result = { ...v }
     extendedSecretFormSchemas.forEach(({ variable }) => {
-      if (result[variable] === formSchemasValue?.[variable])
+      if (result[variable] === formSchemasValue?.[variable] && result[variable] !== undefined)
```

Comment:
When `secret-input` type input is undefined, the web automatically replaces the value with string `'[__HIDDEN__]'`. I disabled the behavior here. Let me know if there is better solution 🙏